test/extensions/filters/http/geoip/geoip_filter_integration_test.cc

```diff
@@ -22,7 +22,7 @@ name: envoy.filters.http.geoip
     typed_config:
       "@type": type.googleapis.com/envoy.extensions.geoip_providers.maxmind.v3.MaxMindConfig
       common_provider_config:
-        geo_headers_to_add:
+        geo_field_keys:
```

Comment:
we should make sure that the old codepath (config fields that are being deprecated) is being tested as well, so either for example do 50/50 split (or some other %) or duplicate the configs.

Reply:
Okay, let me do that.